test/utils/token.test.js

```diff
@@ -0,0 +1,57 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
```

Comment:
```suggestion
require('dotenv').config();
```
Best practice is to require the dotenv module on the first line of the file it's needed in. Also, in this case, you are importing this file but never using it. Try replacing all of the hardcoded 'secrets' in this file :)